pkgs/tools/networking/veilid/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, AppKit
+, Security
+, fetchFromGitLab
+, rustPlatform
+, protobuf
```

Comment:
What is protobuf needed for? I made a dev flake for veilid and so far I've only needed capnproto

Reply:
the opentelemetry crate. Let me try without it.

Reply:
Here's when I remove it:

```
error: failed to run custom build command for `opentelemetry-proto v0.1.0`

Caused by:
  process didn't exit successfully: `/build/source/target/release/build/opentelemetry-proto-8628f340da5d6423/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/common/v1/common.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/trace/v1/trace_config.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.proto
  cargo:rerun-if-changed=src/proto/opentelemetry-proto

  --- stderr
  thread 'main' panicked at 'Could not find `protoc` installation and this build crate cannot proceed without
      this knowledge. If `protoc` is installed and this crate had trouble finding
      it, you can set the `PROTOC` environment variable with the specific path to your
      installed `protoc` binary.If you're on debian, try `apt-get install protobuf-compiler` or download it from https://github.com/protocolbuffers/protobuf/releases

  For more information: https://docs.rs/prost-build/#sourcing-protoc
  ', /build/cargo-vendor-dir/prost-build-0.11.9/src/lib.rs:1457:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: builder for '/nix/store/0jzgnwd1fsvhkbjdphl44y25gy9v05k6-veilid-0.1.7.drv' failed with exit code 101;
       last 10 log lines:
       >   --- stderr
       >   thread 'main' panicked at 'Could not find `protoc` installation and this build crate cannot proceed without
       >       this knowledge. If `protoc` is installed and this crate had trouble finding
       >       it, you can set the `PROTOC` environment variable with the specific path to your
       >       installed `protoc` binary.If you're on debian, try `apt-get install protobuf-compiler` or download it from https://github.com/protocolbuffers/protobuf/releases
       >
       >   For more information: https://docs.rs/prost-build/#sourcing-protoc
       >   ', /build/cargo-vendor-dir/prost-build-0.11.9/src/lib.rs:1457:10
       >   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
       > warning: build failed, waiting for other jobs to finish...
       For full logs, run 'nix log /nix/store/0jzgnwd1fsvhkbjdphl44y25gy9v05k6-veilid-0.1.7.drv'.
```

Reply:
ah you're right! needed for veilid-server. I was just using veilid-core